src/Admin/BaseFieldDescription.php

```diff
@@ -438,8 +464,18 @@ public function mergeOptions(array $options = [])
         $this->setOptions(array_merge_recursive($this->options, $options));
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/admin-bundle 3.x and will be removed in 4.0.
+     */
     public function setMappingType($mappingType)
     {
+        @trigger_error(sprintf(
```

Comment:
Can you verify, that there is no active call from our bundle? Otherwise this will trigger a deprecation in any case

Reply:
> FieldDescriptionInterface::setMappingType looks like it is not being used (it makes sense because the mapping type is being set when the field mapping is set).

I think he already did.

Reply:
Yes, this search shows just calls from tests: https://github.com/search?q=org%3Asonata-project+setMappingType&type=code (I've also checked in orm and mongodb individually)